cloudvolume/lib.py

```diff
@@ -405,7 +405,7 @@ def from_slices(cls, slices, context=None, bounded=False, autocrop=False):
 
     return Bbox(
       [ slc.start for slc in slices ],
-      [ slc.stop for slc in slices ]
+      [ (slc.start if slc.stop < slc.start and (slc.step is None or slc.step > 0) else slc.stop) for slc in slices ]
```

Comment:
I would rather throw an error saying this is unsupported if a step is ever specified. With general convention, people would have expected an subset of the sections/pixels in the case of |step|>1 and sections/pixels in the reverse order in the case of step<0

Reply:
Okay, I'll raise a ValueError.